### PR TITLE
Mattcarter/educator 5607

### PR DESCRIPTION
**Pull request by mattcarter**
**TL;DR -** Rubric assessment sections had excessive vertical whitespace between them. 

JIRA: [EDUCATOR-5607](https://openedx.atlassian.net/browse/EDUCATOR-5607)

**What changed?**
- set margin-bottom to 0px for class

**Developer Checklist**
- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in setup.py and package.json

**Testing Instructions**
Test via devstack. 
 - Checkout this PR, and install it as the local ORA codebase.  
 - Create an ORA and configure it with a rubric 
 - Set 'Show Rubric During Response' to True
 - Navigate to the ORA and view as a learner. Expand the rubric sections.  There should be less vertical whitespace.
 - To confirm navigate the DOM and find the `<li>` element has an effective margin-bottom of 0px

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by nsprenkle** [comment]
Verified locally.

**Before:** Look at all that space! It mocks the page with its greed, callously consuming pixels for naught!

ora-rubric-spacing-before

**After:** Beautiful. 10/10. 

ora-rubric-spacing-after

**Review by nsprenkle — APPROVED** [review]
:shipit:
